src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -3574,6 +3571,7 @@ subroutine RestartBulkOnly ( this, bounds, ncid, flag )
             dim1name='pft', &
             long_name='', units='', &
             interpinic_flag='interp', readvar=readvar, data=this%leafc_to_litter_fun_patch)
+        call set_missing_vals_to_constant(this%leafc_to_litter_fun_patch, 0._r8)
```

Comment:
For this and other new calls to set_missing_vals_to_constant: We (at least I) try to explicitly note where some code is only needed for the sake of backwards compatibility. See comments like this throughout the code:

```Fortran
! BACKWARDS_COMPATIBILITY(wjs, 2016-03-31)
```

This helps us understand what temporary code can possibly be removed a few years down the line. Can you please add these BACKWARDS_COMPATIBILITY notes (perhaps with a pointer to the issue and/or PR) to these new lines?